src/components/Homepage/MonthlyGoodExplained.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+	<div class="monthly-good-explained">
+		<div class="row">
```

Comment:
This div could be combined with the previous one to reduce the number of dom nodes used (good for page performance) `<div class="monthly-good-explained row">`

Reply:
@emuvente Collapsing this into one line. "<div class="row monthly-good-explained"> causes me to lose the ability to color the entire section of the page (to the edges of the page). Currently I  have "monthly-good-explained" background color set to green, which extends to the edges of the page. Adding the row class to this div, prevents the green from running to the edges of the page and contains it to the grid. I'm up to other suggestions, but that's why I did it this way.

Reply:
👍 That makes sense, sorry I didn't notice that css